internal/cmd/login.go

```diff
@@ -525,34 +596,71 @@ func promptLoginOptions(cli *CLI, client *api.Client) (loginMethod loginMethod,
 	}
 }
 
-// Error out if it fails TLS verification and user does not want to connect.
-func promptSkipTLSVerify(cli *CLI) error {
-	// Although the same error, format is a little different for interactive/non-interactive.
-	fmt.Fprintf(os.Stderr, "  %s\n", ErrTLSNotVerified.Error())
-	confirmPrompt := &survey.Confirm{
-		Message: "Are you sure you want to continue?",
+func promptVerifyTLSCert(cli *CLI, cert *x509.Certificate) error {
+	// TODO: improve this message
+	// TODO: use color/bold to highlight important parts
+	// TODO: test format with golden
+	fmt.Fprintf(cli.Stderr, `
+The certificate presented by the server is not trusted by your operating system. It
+could not be automatically verified.
+
+Certificate
+
+Subject: %[1]s
+Issuer: %[2]s
+
+Validity
+  Not Before: %[3]v
+  Not After: %[4]v
+
+Subject Alternative Names:
+  DNS Names: %[5]s
+  IP Addresses: %[6]v
+
+SHA-256 Fingerprint
```

Comment:
Does this get printed in the server logs?

Reply:
It does, but unfortunately only when the cert is first created. 

As part of #2176 I'd like to change cert generation to happen at startup, instead of on first request. If we make that change we can have the fingerprint printed on every startup, instead of only when it is first created.  That will ensure that the value is always in the logs for the pod. I believe right now it'll be hard to find if the server pod was recreated after the cert is generated.

The admin can also generate the the value using:

```
openssl x509 -noout -fingerprint -sha256 -inform pem -in
```

As long as they know where to find the certificate.

Reply:
Makes sense, we can address that in a follow up change that moves cert generation to server start